### ISSUE TITLE
elrs version 3 source code

### ISSUE THREAD

**Issue by iamfaith** [BETAFPV/LiteRadio_PLUS#13]
Thanks for your great project. However, I can not find the source code for elrs 3.0 firmware. It would be great if it is available.

**Comment by chronicle95** [comment]
Agreed. It's a pity the official LR3 firmware with ELRS v3 does not support external modules. It would be great to edit the code to add the support in so I finally have radio that is usable both with and without the Nano TX
